crates/runtime/src/accelerated_table/refresh_task.rs

```diff
@@ -175,9 +182,9 @@ impl RefreshTask {
 
         let dataset_name = self.dataset_name.clone();
 
-        let mode = self.refresh.read().await.mode.clone();
+        let mode = self.refresh.read().await.mode;
 
-        let timer = TimeMeasurement::new(
+        let _ = TimeMeasurement::new(
```

Comment:
Rust will drop the value immediately if you do this - it needs to at least be assigned to a variable so that when it goes out of scope it will be dropped.

Reply:
Updated- thank you